cmd/osbuild-worker/jobimpl-osbuild.go

```diff
@@ -40,16 +40,24 @@ type S3Configuration struct {
 	SkipSSLVerification bool
 }
 
-type OSBuildJobImpl struct {
-	Store             string
-	Output            string
-	KojiServers       map[string]kojiServer
-	GCPCreds          string
-	AzureCreds        *azure.Credentials
-	AWSCreds          string
-	AWSBucket         string
-	S3Config          S3Configuration
+type ContainersConfiguration struct {
 	ContainerAuthFile string
+	Domain            string
+	Account           string
+	CertPath          string
+	TlsVerify         *bool
```

Comment:
`TLSVerify` is more idiomatic, see e.g. https://pkg.go.dev/net/http#Server

Reply:
Done